### ISSUE TITLE
MACRO-10

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1225]
SYS2; TS MACRO

We need MACRO-10 to assemble some programs from the outside.  For example, CROSS #1223.

The binary in SYS2 is pdumped with the DECUUO bootstrap.

**Comment by larsbrinkhoff** [comment]
I don't see that there ever was any MACRO source on ITS, just DECSYS; MACRO SHR.

I tried to use the binary TS MACRO to assemble a random MACRO.MAC I found, but there were many error messages.

**Comment by larsbrinkhoff** [comment]
The SYS2; TS MACRO binary is dated 1976-11-07.  
DECSYS; MACRO SHR is dated 1976-03-14.

Maybe a similarly dated MACRO.MAC file would work?

**Comment by larsbrinkhoff** [comment]
This 1978 version works better: http://pdp-10.trailing-edge.com/ap-c796e-sb/01/macro.mac

```
*macro=macro
PAGE 3
A                                       .DIRECTIVE FLBLST

?1 ERROR DETECTED

HI-SEG. BREAK IS 425771
PROGRAM BREAK IS 003646
CPU TIME USED 00:01.517

11K CORE USED

**Comment by larsbrinkhoff** [comment]
@rcornwell pointed out LINK is a zoo of files.  I added three more, and now LINK works.  I would link the REL file and merge in the DECUUO bootstrap.  Something like this:

```
*:macro
*macro=macro
*^C
*:kill
*:dec sys:link
*macro/go
EXIT
*:start 45
Command: d
*:pdump ts macro

**Comment by larsbrinkhoff** [comment]
Fixed by #1235.
